pkgdb2/api/admin.py

```diff
@@ -151,7 +151,7 @@ def api_admin_actions():
             status=status,
             count=True,
         )
-    except pkgdblib.PkgdbException, err:  # pragma: no cover
+    except pkgdblib.PkgdbException as err:  # pragma: no cover
```

Comment:
The problem was with ```PkgdbBugzillaException``` the ```PkgdbException``` is also availible in pkgdblib, because it is imported there. We can also make it the same in the other places, but it is going to extend the scope of this PR beyond its original goal.